README.md

```diff
@@ -77,8 +78,8 @@ RABBIT_PASSWORD=guest
 SUBSCRIPTION_PROJECT_ID=[SUB_PROJECT_ID]
 RECEIPT_TOPIC_PROJECT_ID=[TOPIC_PROJECT_ID]
 GOOGLE_APPLICATION_CREDENTIALS=[/path/to/service/account/key.json]
-RABBIT_QUEUE=Case.Responses
-RABBIT_EXCHANGE=case-outbound-exchange
```

Comment:
If this exchange is being renamed then we'll need to manually remove it via Rabbit UI for each of our environments. Might be worth adding this as a task to the Trello card? 

Another quick question(!) - will this also mean we'd move from a `direct` to a `topic` exchange?

Reply:
We're using a different exchange 'events', that already exists.  We will be changing the binding from Case.Responses with is currently bound to case-outbound-exchange to events.

Reply:
After this PR is merged in, I think the `case-outbound-exchange` and the queue bindings in Rabbit will stick around in our test environments until they are manually removed. This is because the rabbit instances are continuously running in our test environments and aren't brought down, if that makes sense? I don't think it would hurt leaving the exchanges/bindings there but it might be best to tidy them up as a separate task maybe?